source-browser.html

```diff
@@ -181,7 +209,46 @@ <h1 id="loading" class="m1 center display-none">⌛</h1>
           .focus();
       }
 
+      function createNavbar(topic) {
+        $('.li').append(
+          $('<a>', {
+            id: `li${topic}`,
+            href: `#${topic}`
+          }).text(topic)
+        );
+      }
+
+      let list = [];
+      $(document).ready(() => {
+	    TOPICS.forEach(item => {
+	      let heading = item.name;
+	      list.push(heading);
+	    });
+      });
+
+      $(window).scroll(() => {
+        let top = $(window).scrollTop();
+        let bottom = $(window).scrollTop() + $(window).height();
+        list.forEach(heading => {
+          let vartop = $(`#${heading}`).position();
+          if (top < vartop.top && vartop.top < bottom) {
+            $(`#li${heading}`).hide();
+          } else {
+            $(`#li${heading}`).show();
+            let $heading = $(`#li${heading}`);
+            if (vartop.top > bottom) {
+              $heading.text(`${heading}↓`);
+            } else {
+              $heading.text(`${heading}↑`);
+            }
+          }
+        });
+      });
+
       $(document).ready(() => {
+        $('#navbar').append('<li class="li"></li>');
```

Comment:
Why is there an extra list element?

Reply:
There's only one list element.  Would you like me to make this list element hard-coded in the html instead?

Reply:
Yes

Reply:
Great, I'll do that. :)